02week/blog/css/style.css

```diff
@@ -0,0 +1,123 @@
+
+body {
+  background-color: #27248f;
+}
+
+nav {
+  background-color: #248c30;
+  margin: auto;
+}
+
+p:hover:first-letter {
+  color: teal;
+  text-transform: uppercase;
+  /* font-size: 400px; */
+}
+
+table {
+  margin-left: 30%;
+  margin-right: 30%;
+  table-layout: auto;
+  text-align: left;
+  /* color: #ffffff; */
+}
+
+table, th, td {
+    /* border: 1px solid black; */
```

Comment:
when you're ready to finalize your project, I would suggest removing all the commented out code